noxfile.py

```diff
@@ -248,7 +248,16 @@ def test_docs(session: Session) -> None:
     )
     session.run("sphinx-build", "-b", "doctest", "docs/source", "docs/build")
     # ensure docker image build works too
-    session.run("docker", "build", ".", "--file", "docs/Dockerfile", external=True)
+    # session.run(
+    #    "docker",
+    #    "build",
+    #    ".",
+    #    "--file",
+    #    "docs/Dockerfile",
+    #    "--tag",
+    #   "idom-docs:latest",
+    #    external=True,
+    # )
```

Comment:
I'm not quite sure what's going wrong here. Were you seeing problems with this locally or in CI?

Reply:
I got the same error in local and in CI: `Could not load /app/src/client/node_modules/fast-json-patch/index.mjs`. 

But take a look at line `124`, there is another `docker build` in the `noxfile.py`, so `nox` is building the same docker image 2 times. 

I think the first time `docker build` creates `node_modules` folder in `client` and for some reason, this breaks the second docker build.

Reply:
I'm gonna see what happens if I uncomment this. If it breaks in CI we can investigate.

Reply:
Looks to be working now.

Reply:
OK, then I will need to figure out how to run it locally w/o breaking. So the actual fix here was upgrading npm and fast-json-patch it seems.